app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchScreen.kt

```diff
@@ -116,6 +196,40 @@ class OverdueSearchScreen : BaseScreen<
     overdueSearchRecyclerView.visibility = View.GONE
   }
 
+  override fun showSearchHistory(searchHistory: Set<String>) {
+    overdueSearchHistoryContainer.visibility = View.VISIBLE
+    searchHistoryAdapter.clear()
+    searchHistoryAdapter.addAll(searchHistory)
+  }
+
+  override fun hideSearchResults() {
+    overdueSearchRecyclerView.visibility = View.GONE
+  }
+
+  override fun showOverdueSearchResults(overdueSearchResults: PagingData<OverdueAppointment>) {
```

Comment:
The param name is different from what's defined in the `UiActions` interface. So, there is a lint warning. Let's update the name to resolve it.